mlir/include/mlir/Dialect/MIOpen/MIOpenOps.td

```diff
@@ -206,15 +246,16 @@ def MIOpen_WorkitemIdOp:
   let description = [{
     The `miopen.workgroup_id` op gets the current workitem ID.
   }];
+  let assemblyFormat = "attr-dict `:` type($id)";
 }
 
 // blockwise_copy
 def MIOpen_BlockwiseCopyOp:
     MIOpen_Op<"blockwise_copy">,
     Arguments<(ins AnyMemRef:$source,
                    AnyMemRef:$dest,
-                   VectorOfLengthAndType<[3], [I32]>:$sourceCoordVector,
-                   VectorOfLengthAndType<[3], [I32]>:$destCoordVector,
+                   Fixed1DVector<3, I32>:$sourceCoordVector,
```

Comment:
same as my question above, I don't quite see the merit of going from `VectorOfLengthAndType` to `Fixed1DVector` as of yet. could you help share your insights here?

Reply:
I'm actually working on changing how we use `blockwise_copy` so there could be cases we pass in 3D vectors or 4D vectors. Having this `Fixed1DVector` seems to remove such flexibility offered by `VectorOfLengthAndType`.